tests/kernel/boot_ltp.pm

```diff
@@ -127,9 +127,12 @@ EOF
         script_output($conf_nic_script);
 
         # dhclient requires no wicked service not only running but also disabled
-        script_run(
-            'systemctl --no-pager -p Id show network.service | grep -q Id=wicked.service && { export ENABLE_WICKED=1; systemctl disable wicked; }'
-        );
+        # TODO: chack can be safely removed next release after 20190930 is packaged for QAM
+        if (script_run('grep -q "systemctl restart wicked" /opt/ltp/testcases/bin/dhcp_lib.sh') eq 1) {
+            script_run(
```

Comment:
This 1 line command looks a bit ugly... maybe a suggestion:

    if (systemctl('show network.service', ignore_failure => 1) =~ /Id=wicked.service/) {
        script_run('export ENABLE_WICKED=1');
        systemctl('disable wicked');
    }